### ISSUE TITLE
NFS support

### ISSUE THREAD

**Issue by nizsheanez** [howeyc/fsnotify#107]
Hi. 
go version go1.3.1 linux/amd64 

I runed 2 containers: 
1) with application source code in volume. 
2) with daemon (https://github.com/beego/bee) which listen changes of files in volumes (--volumes-from app_container) 

in volumes (folder mounted by NFS), your library can see when i change files inside container, but when i change it from outside - no notifications.

I found out in https://docs.google.com/document/d/1xl_aRcCbksFRmCKtoyRQG9L7j6DIdMZtrkFAoi5EXaA/edit
that this problem with NFS is known, but "It is unclear how important that case is"

Suggest me please: Must i create ticket about it in Go bug tracker?

**Comment by ghost** [comment]
Yes, NFS doesn't support this type of function.  (And how would it, imagine 20 network computers updating 20 files on one NFS)
